### PR TITLE
Added Aggregator::Base with Celluloid

### PR DESCRIPTION
**Pull request by Hampei**
Added Aggregator.all_aggregators
Added Celluloid.

### CODE REVIEW

File: spec/lib/aggregators/test_spec.rb [hunk 1]
```diff
@@ -0,0 +1,8 @@
+require 'spec_helper'
+
+describe Aggregators::Test do
+  it 'fetch can run as a future' do
+    future = Aggregators::Test.new.future.fetch
+    expect(future.value).to eq []
```
Comment (jobertabma) [inline spec/lib/aggregators/test_spec.rb:6]: I prefer `to be_empty`, reads better when rspec is ran with `-fdoc`

Comment (Hampei) [inline spec/lib/aggregators/test_spec.rb:6]: Ik test niet of ik iets empties terug krijg, ik test of ik een lege array terug krijg. Even heel stom en heel specifiek.

File: lib/aggregators.rb [hunk 2]
```diff
@@ -0,0 +1,10 @@
+Celluloid.shutdown_timeout = 1
+
+Dir[File.dirname(__FILE__) + '/aggregators/*.rb'].each {|file| require file }
+module Aggregators
+
+  def self.all_aggregators
+    Aggregators.constants.select {|c| c != :Base && Class === Aggregators.const_get(c)}
```
Comment (jobertabma) [inline lib/aggregators.rb:7]:
```
Aggregators.constants.map do |constant|
  const = Aggregators.const_get(constant)

  const if const != :Base && Class === const 
end.compact
```

Seems easier to me, but not super opinionated either way. One loop less.

File: spec/lib/aggregators/test_spec.rb (hunk 1 above)
Comment (jobertabma) [inline spec/lib/aggregators/test_spec.rb:6]: Makes sense!

**Comment by jobertabma** [comment]
One minor comment on the loop, but that doesn't block merging this. Feel free to land!

File: lib/aggregators.rb (hunk 2 above)
Comment (Hampei) [inline lib/aggregators.rb:7]: Disagree, more efficient, but less readable.

**Pull request merged**